pkg/apis/config/v1alpha1/resourceinterpreterwebhook_types.go

```diff
@@ -116,6 +116,9 @@ const (
 	// Only necessary for those resource types that have dependencies resources and expect the dependencies be propagated
 	// together, like Deployment depends on ConfigMap/Secret.
 	InterpreterOperationInterpretDependency InterpreterOperation = "InterpretDependency"
+
+	// InterpreterOperationPauseEnsureWork indicates that karmada want to judge whether pause ensure work object or not.
+	InterpreterOperationPauseEnsureWork InterpreterOperation = "PauseEnsureWork"
```

Comment:
https://github.com/karmada-io/karmada/blob/master/docs/userguide/customizing-resource-interpreter.md#built-in-interpreter

Do you need to update the document?